_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm

```diff
@@ -8326,6 +8326,7 @@
 "aDS" = (
 /obj/item/weapon/gun/revolver/cmb,
 /obj/machinery/atmospherics/pipe/simple/green/hidden,
+/obj/effect/landmark/zone_to_control,
```

Comment:
Only ice?

Reply:
that's the only supported map for now, enough for testing